flask_ext_migrate/__init__.py

```diff
@@ -33,33 +33,44 @@ def fix_from_imports(red):
     """
     Converts "from" style imports to not use "flask.ext".
 
-    Handles:
-    Case 1: from flask.ext.foo import bam --> from flask_foo import bam
-    Case 2: from flask.ext import foo --> import flask_foo as foo
+    Handles (with or without parens or linebreaks):
+    from flask.ext.foo import bam --> from flask_foo import bam
+    from flask.ext.foo.bar import bam --> from flask_foo.bar import bam
+    from flask.ext import foo --> import flask_foo as foo
     """
     from_imports = red.find_all("FromImport")
-    for x, node in enumerate(from_imports):
-        values = node.value
-        if len(values) < 2:
+    for node in from_imports:
+        modules = node.value
+
+        if modules[0].value != 'flask' and modules[1].value != 'ext':
             continue
-        if (values[0].value == 'flask') and (values[1].value == 'ext'):
-            # Case 1
-            if len(node.value) == 3:
-                package = values[2].value
-                modules = node.modules()
-                module_string = _get_modules(modules)
-                if len(modules) > 1:
-                    node.replace("from flask_%s import %s"
-                                 % (package, module_string))
+
+        if len(modules) >= 3:
+            tars_str = ''
```

Comment:
Could we come up with a better variable name, here and for other lines where we are using `tar`?